desktop/src/main/java/bisq/desktop/main/overlays/windows/FilterWindow.java

```diff
@@ -301,4 +248,12 @@ private void addContent() {
         gridPane.getChildren().add(hBox);
         GridPane.setMargin(hBox, new Insets(10, 0, 0, 0));
     }
+
+    private List<String> readAsList(InputTextField field) {
+        if (field.getText().isEmpty()) {
+            return new ArrayList<>();
```

Comment:
Use `Collections.emptyList()` instead of `new ArrayList<>()`.

Reply:
`Collections.emptyList()` returns an immutable list, so operations like `add` will throw `UnsupportedOperationException`. Are we sure that these lists are not modified anywhere ?

Reply:
I'm not 100% sure how the result is being used. You can run Bisq with this change and open filter editor and see if anything blows up.

Reply:
Ok, I changed into `Collections.emptyList()` and did like you said - nothing wrong happened.